### PR TITLE
Line edits docs

### PR DESCRIPTION
**Pull request by Hbler**
addresses #96 

## What is included in this PR?
Small line edits in docs.

### CODE REVIEW

File: briefcase-forms.rst [hunk 1]
```diff
@@ -88,24 +88,24 @@ Pulling from Android 4.x and later device
 - Once it is copied onto your local hard drive, unzip the file of the 'odk' directory.
 - Open *ODK Briefcase*.
 - In the window which opens up select :guilabel:`pull` tab.
-- Select *Custom Path to ODK Directory* in the :guilabel:`pull data from` drop-down.
+- Select *Custom Path to ODK Directory* in the :guilabel:`pull data from` the drop-down.
```
Comment (adammichaelwood) [inline briefcase-forms.rst]: i don't understand this addition.

File: briefcase-forms.rst [hunk 2]
```diff
@@ -68,9 +68,9 @@ Pulling from Android 2.x and earlier device
 - Connect your Android device to your computer using a USB cable and choose to mount the drive.
 - Open *ODK Briefcase*.
 - In the window which opens up select :guilabel:`pull` tab.
-- Select `Mounted Android SD Card` in the :guilabel:`pull data from` drop-down.
+- Select `Mounted Android SD Card` in the :guilabel:`pull data from` the drop-down.
```
Comment (adammichaelwood) [inline briefcase-forms.rst]: I don't understand this addition.

File: glossary.rst [hunk 3]
```diff
@@ -80,13 +80,13 @@ Glossary
 
   widget
 
-    A single question, answer set, and attendant GUI elements, as rendered in an XForm compliant app such as :term:`Collect`.
+    A single question, answer set, and attendant GUI elements, as rendered in a XForm compliant app such as :term:`Collect`.
```
Comment (adammichaelwood) [inline glossary.rst]: "XForm" pronounced *eks-form*, so "an XForm" is correct.

File: glossary.rst [hunk 4]
```diff
@@ -80,13 +80,13 @@ Glossary
...
   hint
 
     Additional help text on a single question, displayed after the label.
 
   question label
   question text
-  
-    The main part body of a form question or widget. In an :term:`XLSForm`, the contents of :th:`label`.
+
+    The main part body of a form question or widget. In a :term:`XLSForm`, the contents of the :th:`label`.
```
Comment (adammichaelwood) [inline glossary.rst]: "XForm" is pronounced *eks-form*, so "an XForm" is correct.

File: briefcase-forms.rst (hunk 2 above)
Comment (Hbler) [reply briefcase-forms.rst]: Yes, I misinterpreted the text. I'll correct it!

File: briefcase-forms.rst (hunk 1 above)
Comment (Hbler) [reply briefcase-forms.rst]: Yes, I misinterpreted the text. I'll correct it!

File: glossary.rst (hunk 3 above)
Comment (Hbler) [reply glossary.rst]: Thanks for pointing it out, I was so focused on the grammatical aspect that I oversaw the phonetical part.

File: glossary.rst (hunk 4 above)
Comment (Hbler) [reply glossary.rst]: Thanks for pointing it out, I was so focused on the grammatical aspect that I oversaw the phonetical part.

**Pull request merged**